### DISCUSSION TITLE
A way to not to show dock/project panel by default?

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#6496] [Feature Requests]
When opening a new window, the dock is shown by default;
when switching to a known project the project panel pane is shown;
when opening  a new project, both dock and the project panel are shown. 

Is there a way to not to show the dock and the pane in these situations? I would like to toggle them later when necessary. Thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
